data/js/values/content1.js

```diff
@@ -1,5 +1,5 @@
 (function() {
-    var answer = document.querySelector('#answer');
+    var answer = document.querySelector('#answer span');
```

Comment:
Highlighting of only the text looks a bit clunky. It would help if we had a UX image to design off of. Maybe this is something we can revisit in London.